src/Mvc/Mvc.Core/src/ModelBinding/Validation/ValidationVisitor.cs

```diff
@@ -234,6 +275,13 @@ protected virtual bool ValidateNode()
             }
         }
 
+        /// <summary>
+        /// Validate something in a model.
+        /// </summary>
+        /// <param name="metadata">The model metadata.</param>
+        /// <param name="key">The key to validate.</param>
+        /// <param name="model">The model to validate.</param>
+        /// <returns>Whether the the specified model key is valid.</returns>
```

Comment:
/// <returns><see langword="true" if the specified model key is valid, otherwise <see langword="false" />.</returns>

Reply:
some of the docs use `<c>true</c>` do we care about consistency?

Reply:
If you'd like, you could do a find and replace.

Reply:
Which do we prefer using?

Reply:
The langword one